content/api/cypress-api/custom-commands.md

```diff
@@ -685,6 +685,12 @@ to those commands, and programmatically controlling your custom command. This
 will cleanup the Command Log and be much more visually appealing and
 understandable.
 
+### `cy.hover()` and `cy.mount()`
+
+Cypress does not have hover() or mount() commands out-of-the-box. See how to
+craft your own [cy.hover()](/api/commands/hover) and
+[cy.mount()](/api/commands/mount) custom commands.
```

Comment:
```suggestion
### `cy.hover()` and `cy.mount()`

Cypress does not have `cy.hover()` or `cy.mount()` commands out-of-the-box. See how to
craft your own [`cy.hover()`](/api/commands/hover) and
[`cy.mount()`](/api/commands/mount) custom commands.
```

Reply:
Why is this marked as resolved? It doesn't look like the changes were made.

Reply:
oops, may have been trying to squash the previous comment.